project-keeper/src/main/resources/maven_templates/maven-enforcer-plugin.xml

```diff
@@ -11,7 +11,7 @@
             <configuration>
                 <rules>
                     <requireMavenVersion>
-                        <version>3.6.3</version>
+                        <version>3.8.7</version>
```

Comment:
```suggestion
                        <version>[3.8.7,3.9.0)</version>
```
❓  to force `3.8.7 <= version < 3.9.0`, see https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html